ci/launch.sh

```diff
@@ -15,14 +15,32 @@ uploadAssembly "${ASSEMBLY_PREFIX}" "${PROJECT_NAME}"
 markStartOfBlock "Launching deployments"
 
 dotnet run -p workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/DeploymentLauncher.csproj -- \
-    create "${PROJECT_NAME}" "${ASSEMBLY_NAME}" \
-    "${ASSEMBLY_NAME}" cloud_launch_large.json snapshots/cloud.snapshot EU \
-    "${ASSEMBLY_NAME}_sim_players" cloud_launch_large_sim_players.json
+    create \
+    --project_name "${PROJECT_NAME}" \
+    --assembly_name "${ASSEMBLY_NAME}" \
+    --deployment_name "${ASSEMBLY_NAME}" \
+    --launch_json_path cloud_launch_large.json \
+    --snapshot_path snapshots/cloud.snapshot \
+    --region EU \
+    --tags "dev_login"
 
 CONSOLE_URL="https://console.improbable.io/projects/${PROJECT_NAME}/deployments/${ASSEMBLY_NAME}/overview"
-CONSOLE_URL_SIM_PLAYERS="https://console.improbable.io/projects/${PROJECT_NAME}/deployments/${ASSEMBLY_NAME}_sim_players/overview"
 
 buildkite-agent annotate --style "success" "Deployment URL: ${CONSOLE_URL}<br/>"
+
+dotnet run -p workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/DeploymentLauncher.csproj -- \
+    create-sim \
+    --project_name "${PROJECT_NAME}" \
+    --assembly_name "${ASSEMBLY_NAME}" \
+    --deployment_name "${ASSEMBLY_NAME}_sim_players" \
+    --launch_json_path cloud_launch_large_sim_players.json \
+    --region EU \
+    --target_deployment "${ASSEMBLY_NAME}" \
+    --flag_prefix fps \
+    --simulated_coordinator_worker_type SimulatedPlayerCoordinator
```

Comment:
`simulated_coordinator_worker_type` --> `sim_player_coordinator_worker_type` or `simplayer_coordinator_worker_type` 🙏 

(probs another PR)

Reply:
Yeah will do once #171 and this PR are merged to reduce conflicts / Git pain